sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/FileIndexSuite.scala

```diff
@@ -95,7 +96,7 @@ class FileIndexSuite extends SharedSQLContext {
       withSQLConf(SQLConf.CASE_SENSITIVE.key -> "false") {
         val fileIndex = new InMemoryFileIndex(spark, Seq(path), Map.empty, None)
         val partitionValues = fileIndex.partitionSpec().partitions.map(_.values)
-        assert(partitionValues.length == 2)
+        assert(partitionValues.length === 2)
```

Comment:
can you revert these unnecessary changes?

Reply:
I think this change is fine. It explicitly uses the scalatest macro. I recently learned that will also print a useful message for `==`, though I kind of like using the explicit macros

Reply:
Hi @cloud-fan, just as @srowen mentioned. `===` should print more message than `==`.

Reply:
That's not always the case (see https://github.com/databricks/scala-style-guide/issues/36). I heard that it can compare arrays properly as well. It would be great if we can document it there.

Reply:
thanks @HyukjinKwon, will change back.